.travis.yml

```diff
@@ -7,13 +7,16 @@ install:
   - pip install sphinx
   - pip install sphinx_rtd_theme
   - pip install numpydoc
+  - pip install jupyter jupyter_client nbconvert nbval
 
 script:
   # Use Sphinx to make the html docs
   - cd docs
   - make html
   - touch _build/html/.nojekyll
-    
+  # roughtly execute every jupyter notebook we fine
+  - find . -name requirements.txt|xargs cat|xargs pip install -U 
```

Comment:
What do you want to achieve with this line?

Reply:
I've replaced this now, see comment below